src/index.html

```diff
@@ -11,6 +11,33 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      action="/"
```

Comment:
Right now I get this 405 page when I press enter, fix your action value
![image](https://user-images.githubusercontent.com/37161315/137087088-bc8618e3-ca73-4a1f-9a35-24a7e21302d0.png)